Translators/WZDx/geospatial_service.py

```diff
@@ -4,7 +4,21 @@
 import urllib3
 urllib3.disable_warnings()
 from Schemas import geospatial_schemas as geospatial_schemas
+import redis
+import json
 
+cache = None
+
+def get_redis_connection():
+    global cache
+    if cache is None:
+        cache = redis.Redis(host=os.environ['REDIS_HOST'], port=os.environ['REDIS_PORT'], decode_responses=True, db=0, password=os.environ['REDIS_PASS'])

```

Comment:
Let's add these new environment variables to the sample.env and the README

Reply:
Added to sample.env and updated README to include redis documentation and these variables